src/Prelude/Internal.purs

```diff
@@ -0,0 +1,39 @@
+module Prelude.Internal where
+
+-- | Applies a function to its argument.
+-- |
+-- | ```purescript
+-- | length $ groupBy productCategory $ filter isInStock $ products
+-- | ```
+-- |
+-- | is equivalent to:
+-- |
+-- | ```purescript
+-- | length (groupBy productCategory (filter isInStock products))
+-- | ```
+-- |
+-- | `($)` is different from [`(#)`](#-2) because it is right-infix instead of
```

Comment:
No idea, it was copy/pasted. I should probably update the comment to reflect the named functions anyway?